build.properties

```diff
@@ -12,5 +12,5 @@ wpilib.pluginURL=http://first.wpi.edu/FRC/roborio/release/eclipse/plugins/edu.wp
 
 # If set, and set to true or yes, will download and install the CTRE User Library
 userLibInstallCtre=true
-userLibInstallCtre.URL=http://www.ctr-electronics.com/downloads/lib/CTRE_FRCLibs_NON-WINDOWS.zip
+userLibInstallCtre.URL=http://www.ctr-electronics.com/downloads/lib/CTRE_FRCLibs_NON-WINDOWS_v4.4.1.9.zip
```

Comment:
99% sure. This is the version that doesn't come with the windows installer, and just lets you get files that you can place manually.